File: abc/abc158/a.cpp
// https://atcoder.jp/contests/abc158/submissions/26250197

#include <bits/stdc++.h>
using namespace std;
int main() {
  char a, b, c;
  cin >> a >> b >> c;
  if((a == b) && (b == c) && (c == a)) cout << "No";
  else cout << "Yes";
}

File: abc/abc158/a.java
// https://atcoder.jp/contests/abc158/submissions/43246448

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    String s = stdIn.next();
    if(s.equals("AAA") || s.equals("BBB")) System.out.println("No");
    else System.out.println("Yes");
  }
}
